pkg/kubeapiserver/options/authentication.go

```diff
@@ -622,9 +623,10 @@ func (o *BuiltInAuthenticationOptions) ApplyTo(authInfo *genericapiserver.Authen
 		authenticatorConfig.CustomDial = egressDialer
 	}
 
-	authInfo.Authenticator, openAPIConfig.SecurityDefinitions, err = authenticatorConfig.New()
+	var openAPIV3SecuritySchemes spec3.SecuritySchemes
+	authInfo.Authenticator, openAPIConfig.SecurityDefinitions, openAPIV3SecuritySchemes, err = authenticatorConfig.New()
 	if openAPIV3Config != nil {
-		openAPIV3Config.SecurityDefinitions = openAPIConfig.SecurityDefinitions
+		openAPIV3Config.SecuritySchemes = openAPIV3SecuritySchemes
 	}
 	if err != nil {
```

Comment:
check this first and return before assigning the result of authenticatorConfig.New() to input variables